.github/update_conda_lock.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+from os import environ
+from subprocess import run, DEVNULL, PIPE
+from sys import exit
+from re import match, search
+
+def _run(cmd_string, multiword_last_arg='', return_stdout=False, **kwargs):
+    cmd = cmd_string.split() + ([multiword_last_arg]
+            if multiword_last_arg else [])
+    if return_stdout:
+        return run(cmd, check=True, encoding='utf-8', stdout=PIPE,
+                **kwargs).stdout
+    else:
+        return run(cmd, check=True, **kwargs)
+
+def _get_env(env_name, required=True):
+    if env_name not in environ:
+        if required:
+            print('ERROR: Required environment variable not found: ' + env_name + '!')
```

Comment:
Why does this print, versus throw an exception?  Seems like an error is an error?

Reply:
I didn't want to exit in this precise moment to print all required variables that are missing.